endpoints.json

```diff
@@ -61,5 +61,10 @@
         "created_at": "2020-06-20T07:24:00.000Z"
       }
     ]}
-  }
+  },
+  "POST /api/articles/:article_id/comments": {
```

Comment:
Make sure you also include what the post body needs to take (in this case, a key of author and a body)